apps/frontend/src/libs/components/tab-container/tab-container.tsx

```diff
@@ -0,0 +1,68 @@
+import { combineClassNames } from "~/libs/helpers/helpers.js";
+import { useState } from "~/libs/hooks/hooks.js";
+
+import { type TabContainerProperties } from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+const TabContainer = ({
+	activeTabClassName,
+	containerClassName,
+	contentClassName,
+	defaultTabId,
+	navigationClassName,
+	tabClassName,
+	tabs,
+}: TabContainerProperties): React.JSX.Element => {
```

Comment:
We need to avoid passing classnames to common ui components (that already have predefined styles), to have all possible combinations/looks fixed in the component itself, and avoid unexpected side-effects across the app.  If any adjustment is needed, e.g., small/big, style - primary/secondary - it should be done explicitly through props and fixed interface. In our case, I believe we don't need all of this, and we only need items, while everything else can be defined here as part of the common logic
```suggestion
const Tabs = ({
    activeTabId,
	items,
}: Properties): React.JSX.Element => {
```

We also need to create a hook, something like useTabsNavigation, which will be used outside of this component, and set appropriate query params/id to url and detect which tab is currently active by URL. This hook will return activeTabId or something like this, which we will pass to this component